project/deps.sc

```diff
@@ -139,7 +139,7 @@ object Deps {
   def ammoniteForScala3Lts = ivy"com.lihaoyi:::ammonite:${Versions.ammoniteForScala3Lts}"
   def asm                  = ivy"org.ow2.asm:asm:9.7.1"
   // Force using of 2.13 - is there a better way?
-  def bloopConfig = ivy"ch.epfl.scala:bloop-config_2.13:2.0.3"
+  def bloopConfig = ivy"ch.epfl.scala:bloop-config_2.13:2.1.0"
```

Comment:
Actually, we could now switch to Scala 3 artifact. Do we want that @Gedochao ?

Reply:
I think we do. 
Let's do that in a separate PR.